client/src/components/History/Content/Collection/CollectionProgress.vue

```diff
@@ -3,11 +3,15 @@ at components/JobStates/CollectionJobStates but it relies on the backbone data
 model, so probably has to go eventually.-->
 <template>
     <div class="collection-progress">
-        <b-progress v-if="maxJobs && runningJobs" :max="maxJobs" show-value>
-            <b-progress-bar v-if="errorJobs" :value="errorJobs" variant="danger" />
-            <b-progress-bar v-if="okJobs" :value="okJobs" variant="success" />
-            <b-progress-bar v-if="runningJobs" :value="runningJobs" variant="warning" />
-            <b-progress-bar v-if="waitingJobs" :value="waitingJobs" variant="secondary" />
+        <b-progress v-if="maxJobs != okJobs + errorJobs" :max="maxJobs">
+            <b-progress-bar v-if="errorJobs" :value="errorJobs" variant="danger" v-b-tooltip.hover="errorJobs" />
+            <b-progress-bar v-if="okJobs" :value="okJobs" variant="success" v-b-tooltip.hover="okJobs" />
+            <b-progress-bar v-if="runningJobs" :value="runningJobs" variant="warning" v-b-tooltip.hover="runningJobs" />
```

Comment:
I think you're missing paused and deleted jobs here ?

Reply:
I don't see deleted in here `client/src/components/History/Content/model/states.js` 

Should paused be the same as waiting jobs? Or another color?

Reply:
Some confusion over whether the state is DELETED (as noted in `JobStateSummary`) or discarded as defined in `states.js`?

There is no deleted state in `states.js`.